sale_ux/views/sale_order_views.xml

```diff
@@ -99,4 +99,15 @@
         <field name="groups_id" eval="[(6, 0, [ref('base.group_no_one')])]"/>
     </record>
 
+    <record id="sale_order_pricelist_form_view_inh" model="ir.ui.view">
```

Comment:
Here this should be in this record view https://github.com/adhoc-dev/sale/blob/c90c2368760e4e9fe9989f0e853192b6aadec5e7/sale_ux/views/sale_order_views.xml#L4